etc/requirements/base.txt

```diff
@@ -27,12 +27,12 @@ gunicorn==20.1.0
 container_inspector==21.6.10
 
 # ScanCode-toolkit
-scancode-toolkit[packages]==21.6.7
+scancode-toolkit[packages]==21.6.11
 extractcode[full]==21.6.2
-commoncode==21.5.25
+commoncode==21.6.11
 
 # FetchCode
-fetchcode-container==1.2.3.210512; sys_platform == "linux"
+fetchcode-container>=1.2.3.210512; sys_platform == "linux"
```

Comment:
Let's use pinned down version instead ;)

Reply:
I've pinned the dep.